libcrux-ml-dsa/src/simd/avx2/ntt.rs

```diff
@@ -16,6 +51,9 @@ fn butterfly_2(
     zeta_b2: i32,
     zeta_b3: i32,
 ) {
+    let re0 = re[index].value;
```

Comment:
The idea here is to not have the array index in the computation? Please add a comment (here and the other places this happens) that it's better this way for the proof.

Reply:
added a note.